lib/tpm2_util.c

```diff
@@ -41,6 +41,9 @@
 #include "tpm2_tool.h"
 #include "tpm2_util.h"
 
+#define FILE_PREFIX "file:"
+#define FILE_PREFIX_LEN strlen(FILE_PREFIX)
```

Comment:
this should be (sizeof(FILE_PREFIX) - 1) to take advantage of known size at compile time. But I won't nack this series for that. It's a minor optimization.